projects/models.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  pkgs,
+  sources,
+}:
+let
+  yants = import sources.yants { };
+
+  inherit (yants)
+    string
+    list
+    option
+    attrs
+    enum
+    either
+    ;
+in
+rec {
+  project =
+    p: with p; {
+      name = string name;
+      metadata = with metadata; {
+        summary = string summary;
+        funds = list (enum [
+          "Commons"
+          "Core"
+          "Entrust"
+          "Review"
+        ]) funds;
+        websites = attrs (either (option string) (list string)) {
+          repo = string websites.repo;
+          docs = option string (websites.docs or null);
+          blog = option string (websites.blog or null);
+          forum = option string (websites.forum or null);
+          matrix = option string (websites.matrix or null);
+          other = list string (websites.other or [ ]);
+        };
+        contact = {
+          email = option string (contact.email or null);
+        };
```

Comment:
It's alright to keep the field if we don't expose that information from the NLnet dashboard. In any case I suggest not adding that if we don't have a concrete use case for it.